backend/src/main/resources/config/liquibase/changelog/20230531185949_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="kevinmessiaen (generated)" id="1685552392648-1">
+        <dropDefaultValue columnDataType="boolean" columnName="cell_level" tableName="callable_functions"/>
+    </changeSet>
+    <changeSet author="kevinmessiaen (generated)" id="1685552392648-2">
+        <dropDefaultValue columnDataType="varchar(50)" columnName="created_by" tableName="inspections"/>
+    </changeSet>
+    <changeSet author="kevinmessiaen (generated)" id="1685552392648-3">
+        <dropDefaultValue columnDataType="int" columnName="process_type" tableName="callable_functions"/>
+    </changeSet>
+    <changeSet author="kevinmessiaen (generated)" id="1685552392648-4">
+        <dropDefaultValue columnDataType="boolean" columnName="sample" tableName="inspections"/>
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Is this one all good ? I don't see changes in the code to the inspections table, but see some here?

Reply:
Actually it's not even in the master.xml

Reply:
Yes you're right I comited it by mistake, I removed it